Makefile-az.mk

```diff
@@ -25,6 +25,32 @@ az-all-custom-vnet: az-login az-create-workload-msi az-mkaks-custom-vnet az-crea
 
 az-all-savm:    az-login az-mkaks-savm az-perm-savm az-patch-skaffold-azure az-build az-run az-run-sample ## Provision the infra (ACR,AKS); build and deploy Karpenter; deploy sample Provisioner and workload - StandaloneVirtualMachines
 
+install-tools: az-tool ## install tools to create a local developer environment
```

Comment:
Maybe consider a separate script (under hack) to be called from makefile?
Consider adding `shellcheck` and `pre-commit`.

Reply:
`shellcheck` was referring to this tool: https://www.shellcheck.net/; the name of the script in hack needs to be something else. (Maybe devtools.sh? A little hard to express the difference between this and toolchain.sh ...)